src/expire.c

```diff
@@ -302,6 +308,13 @@ void activeExpireCycle(int type) {
                  (expired*100/sampled) > config_cycle_acceptable_stale);
     }
 
+    /* Propagate all DELs */
+    propagatePendingCommands();
```

Comment:
aren't we missing a similar call in the eviction (tenacity) timer path?

i don't recall if there's a test that triggers this path in a master/replica configuration.
maybe it's a good idea to add one (or extend an existing one).

Reply:
we have the same call in performEvictions which is called either from the timer or processCommand

Reply:
sorry.. somehow i missed it.
but maybe add a test for eviction and propagation anyway? (in case we don't have one)?